### PR TITLE
[zh_CN]: Update translations

### PR DESCRIPTION
**Pull request by zeturn**
Some of the previous translations were literal translations of words, which Chinese users could not understand

### CODE REVIEW

File: locales/zh_CN/php.json [hunk 1]
```diff
@@ -135,19 +135,19 @@
     "size.file": ":Attribute 大小必须为 :size KB。",
     "size.numeric": ":Attribute 大小必须为 :size。",
     "size.string": ":Attribute 必须是 :size 个字符。",
-    "slug_input_explanation": "此 slug 用于在 URL 中识别您的团队。",
+    "slug_input_explanation": "此slug短标签用于在 URL 中识别您的团队。",
```
Comment (overtrue) [inline locales/zh_CN/php.json:138]:
```diff
- 此slug短标签用于在
+ 此 slug 短标签用于在
```

**Review by overtrue — CHANGES_REQUESTED** [review]

File: locales/zh_CN/php.json (hunk 1 above)
Comment (andrey-helldar) [reply locales/zh_CN/php.json:138]: @zeturn, can you fix that?

Comment (overtrue) [reply locales/zh_CN/php.json:138]: @andrey-helldar maybe you can edit it directly and then merge it?

Comment (andrey-helldar) [inline locales/zh_CN/php.json:138]:
```suggestion
    "slug_input_explanation": "此 slug 短标签用于在 URL 中识别您的团队。",
```
@overtrue, yes, I can.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Review by andrey-helldar — CHANGES_REQUESTED** [review]
谢谢！

**Pull request merged**